components/hab/Cargo.toml

```diff
@@ -17,6 +17,10 @@ env_logger = "*"
 futures = "*"
 # Pending upgrade activities in https://github.com/habitat-sh/core/issues/72
 hyper = "0.10"
+habitat_api_client = { path = "../builder-api-client" }
```

Comment:
🤔 We should think about renaming this (either the crate or the directory); I thought this was a copy-paste error at first!

Reply:
Agreed, it's not super intuitive.